tests/Mentat.Domain.Tests/BashTestDriver.Tests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Xunit;
+using Autofac.Extras.Moq;
+using Mentat.Domain.Interfaces;
+using Mentat.Domain.Bash;
+using FluentAssertions;
+using Moq;
+using System.Linq;
+
+namespace Mentat.Domain.Tests
+{
+    public class UnitTest1
```

Comment:
@lisa000s can you make a commit changing this class name so it matches the file name?

Reply:
Actuall let's rename from UnitTest1 to BashTestDriverTests (doesn't need to match file name perfectly)